src/main/java/seedu/address/storage/JsonSerializableAddressBook.java

```diff
@@ -37,16 +42,24 @@ public JsonSerializableAddressBook(@JsonProperty("persons") List<JsonAdaptedPers
      * @param source future changes to this will not affect the created {@code JsonSerializableAddressBook}.
      */
     public JsonSerializableAddressBook(ReadOnlyAddressBook source) {
-        persons.addAll(source.getPersonList().stream().map(JsonAdaptedPerson::new).collect(Collectors.toList()));
+        persons.addAll(source.getPersonList().stream().map(JsonAdaptedPerson::new).toList());
```

Comment:
Just out of curiosity, is there a difference between doing `.collect(Collectors.toList())` and `toList()`

Reply:
Not that I know of, the IDE just recommended making the change and I was tired of seeing the squiggly yellow line 😆